json/pt_BR.json

```diff
@@ -1,64 +1,169 @@
+    "Switch Teams": "Mudar de time",
+    "Team Details": "Detalhes do time",
+    "Team Members": "Membros do time",
+    "Team Name": "Nome do time",
+    "Team Owner": "Dono do time",
+    "Team Settings": "Configurações do time",
     "The :attribute must be a valid role.": ":attribute deve ser um papel válido.",
+    "The :attribute must be at least :length characters and contain at least one number.": "O campo :attribute deve possuir no mínimo :length caracteres e conter pelo menos um número.",
+    "The :attribute must be at least :length characters and contain at least one special character.": "O campo :attribute deve possuir no mínimo :length caracteres e conter pelo menos um caractere especial.",
+    "The :attribute must be at least :length characters and contain at least one uppercase character and one number.": "O campo :attribute deve possuir no mínimo :length caracteres e conter pelo menos uma letra maiúscula e um número.",
+    "The :attribute must be at least :length characters and contain at least one uppercase character and one special character.": "O campo :attribute deve possuir no mínimo :length caracteres e conter pelo menos uma letra maiúscula e um caractere especial.",
+    "The :attribute must be at least :length characters and contain at least one uppercase character, one number, and one special character.": "O campo :attribute deve possuir no mínimo :length caracteres e conter pelo menos uma letra maiúscula, um número, e um caractere especial.",
+    "The :attribute must be at least :length characters and contain at least one uppercase character.": "O campo :attribute deve possuir no mínimo :length caracteres e conter pelo menos uma letra maiúscula.",
+    "The :attribute must be at least :length characters.": "O campo :attribute deve possuir no mínimo :length caracteres.",
+    "The provided password does not match your current password.": "A senha informada não corresponde à sua senha atual.",
+    "The provided password was incorrect.": "A senha informada está incorreta.",
+    "The provided two factor authentication code was invalid.": "O código de autenticação em dois fatores informado não é válido.",
+    "The team's name and owner information.": "Nome do time e informações do dono.",
     "This action is unauthorized.": "Esta ação não é autorizada.",
+    "This device": "Este dispositivo",
     "This password does not match our records.": "Esta senha não corresponde aos nossos registros.",
     "This password reset link will expire in :count minutes.": "O Link de redefinição de senha irá expirar em :count minutos.",
     "This user already belongs to the team.": "Este usuário já pertence a um time.",
+    "to": "para",
```

Comment:
Please review the pagination context I believe I should be "até" as in "Mostrando 4 até 10 de 200 resultados"

Here's the original code [(https://github.com/laravel/framework/blob/e63abd8d372f9bad3ca95570cb7e603f3ef28bdc/src/Illuminate/Pagination/resources/views/tailwind.blade.php]((https://github.com/laravel/framework/blob/e63abd8d372f9bad3ca95570cb7e603f3ef28bdc/src/Illuminate/Pagination/resources/views/tailwind.blade.php)